### ISSUE TITLE
Please add option in dashboard to disable ads when admin user is logged in

### ISSUE THREAD

**Issue by sachink23** [google/site-kit-wp#828]
## Feature Description
As we all know Google disables AdSense accounts when it detects invalid click or invalid impressions, to avoid these types of clicks and impressions, google site kit team should add an option in the dashboard of the site kit plugin to disable ads on the website when the admin, writer, editor or moderator is signed in to the wp-admin, so that it will help content creators while using plugins like Elementor for editing a previous post or while creating a new post.

In short, what I want is to remove the ad placement code placed by the google site kit plugin for the authorized users like admin, writer, editor or moderator. Only keep the code if logged in user is a subscriber or user is not logged in.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## Changelog entry

*

**Comment by aaemnnosttv** [comment]
This is essentially an AdSense version of #88

**Comment by sachink23** [comment]
Yes, @aaemnnosttv just seen #88, addressing these issues will help site admins for better tracking of analytics and implementation of adsense.

**Comment by felixarntz** [comment]
This is now being prioritized and tracked via #2681, i.e. I'll close this one as a duplicate now.
